astro/src/content/articles/identity-basics/authorization-models.mdx

```diff
@@ -0,0 +1,134 @@
+
+## ABAC: Determining the Who, What, When, Where, and Why in Access Control
+
+[Attribute-Based Access Control (ABAC)](https://www.youtube.com/watch?v=cgTa7YnGfHA) establishes policies based on a combination of attributes that define what actions can be performed. It controls access based on details of a request and contextual factors, asking who can have access to what and under what conditions. Understanding how attributes are used in ABAC is crucial for distinguishing it from Policy-Based Access Control (PBAC), which we'll discuss next. Remember that ABAC relies on a **predefined list of attributes**. There are four categories of attributes used to make decisions:
+
+
+
+* **Subject **attributes describe the characteristics of users or entities requesting access. These attributes include their name, age, job title, department, group affiliation, and management level. A role can be a subject attribute, but, in contrast to RBAC, it won't completely determine access. For example, Roy and Moss in the IT department require access to general IT information. While both need similar permissions, Moss is a compliance officer with additional responsibilities, while Roy is a regular user. In an ABAC system, a range of attributes beyond roles are considered. User attributes such as 'security clearance' and 'special responsibilities' play a crucial role. Moss's higher security clearance and additional responsibilities allow him to access more sensitive information than Roy.
+* **Resources** refer to the items that someone wants or needs access to, such as a file or a web application. Resource attributes include identifying characteristics of the resource, such as the owner, name, type, creation time, and data sensitivity. To build on the previous example, while both Roy and Moss can see the owner and name of a file, if a file contains sensitive information, only Moss can see the contents. This is because the resource attribute 'data sensitivity' requires higher clearance and special responsibilities, which Moss possesses. 
+* **Action **characteristics dictate how users can interact with requested resources, with common actions including viewing, reading, writing, editing, sharing, and deleting. For example, Moss can have permissions to read and write a client's file, but the authority to delete is reserved for Jen. Instead of just giving broad access, ABAC can specify exactly what each user is allowed to do (e.g., read, write, delete) based on their attributes and the attributes of the resources.
```

Comment:
```suggestion
* **Action** characteristics dictate how users can interact with requested resources, with common actions including viewing, reading, writing, editing, sharing, and deleting. For example, Moss can have permissions to read and write a client's file, but the authority to delete is reserved for Jen. Instead of just giving broad access, ABAC can specify exactly what each user is allowed to do (e.g., read, write, delete) based on their attributes and the attributes of the resources.
```